shell/dialog/AddCustomBadgeDialog.vue

```diff
@@ -312,15 +312,23 @@ export default {
         />
         <div class="buttons">
           <button
+            role="button"
             class="btn role-secondary mr-10"
+            :aria-label="t('generic.cancel')"
             @click="close"
+            @keyup.enter="close"
+            @keyup.space="close"
```

Comment:
`enter` and `space` should trigger the click event for native button elements. Do we need to specify these here?